sdk/identity/ci.yml

```diff
@@ -28,8 +28,8 @@ stages:
       ServiceDirectory: identity
       CtestRegex: azure-identity-unittest.
       LiveTestCtestRegex: azure-identity-livetest.
-      LineCoverageTarget: 99
-      BranchCoverageTarget: 62
+      LineCoverageTarget: 100
+      BranchCoverageTarget: 100
```

Comment:
🎉 

Are we sure we want to keep this as the bar? Seems really high to maintain without putting a bunch of (seemingly) arbitrary annotations. ~95 allows us that flexibility.

I am curious to know what you and others think.

Reply:
I don't disagree, especially 100 won't make it possible to simply submit a draft PR without dropping these numbers temporarily. But also to be fair, this might be the problem with any number - 95, 90, if you add a lot o code.

Under the approach in this PR, where you mark everything unactionable (noise) with annotations, 100% is actually a good result, meaning that everything has been addressed. The moment something drops to 99%, you know there is higher chance of a real coverage problem. Every line you'd see in red would mean it is not covered, without the inherent mysterious 5% coverage gap in the system that we admit we can't do anything about.